02/ht/example/script.js

```diff
@@ -110,12 +196,40 @@ function log(x) {
  * https://github.com/MostlyAdequate/mostly-adequate-guide-ru/blob/master/ch4-ru.md
  * @param {*} func
  */
-function curry(func) {}
+
+function target1(a, b, c, d) {
+  return a + b + c + d;
+}
+
+function curry(func) {
+  let arr = [];
+  let res = 0;
+  function check(a) {
```

Comment:
и снизу `return check` можно будет убрать
```suggestion
  return function check(a) {
```